iot_config/src/route_service.rs

```diff
@@ -413,13 +404,7 @@ impl iot_config::Route for RouteService {
             tracing::info!("existing routes sent; streaming updates as available");
             telemetry::route_stream_subscribe();
             loop {
-                let shutdown = shutdown_listener.clone();
```

Comment:
this can also avoid the select macro and probably just have a `while let Ok(update) = stream.recv() {}` inside the loop

Reply:
addressed here: ec80d34b0873a8a45fb277efa7f5b64b8cb28bf8